src/components/PlayerCard/types.ts

```diff
@@ -0,0 +1,6 @@
+export type Props = {
+    color?: string;
+    name?: string;
+    playerType?: string;
+    playerHealth?: string;
+  };
```

Comment:
❓
Тут не улетело форматирование?
```suggestion
  color?: string;
  name?: string;
  playerType?: string;
  playerHealth?: string;
};
```